frontend/src/metabase/app.js

```diff
@@ -14,70 +14,108 @@ import 'angular-http-auth';
 
 import "./controllers";
 import "./directives";
-import "./filters";
-import "./forms";
 import "./services";
 
-import "./icons";
 
-import "./auth/auth.module";
-import "./home/home.module";
-import "./pulse/pulse.module";
-import "./query_builder/query_builder.module"
-import "./setup/setup.module";
-import "./user/user.module";
+import React from "react";
+//import { render } from "react-dom";
+//import { Provider } from "react-redux";
+import { applyMiddleware, combineReducers, compose, createStore } from "redux";
+import { reducer as form } from "redux-form";
+import createLogger from "redux-logger";
+import promise from 'redux-promise';
+import { reduxReactRouter, routerStateReducer } from "redux-router";
+import thunk from "redux-thunk";
+import { createHistory } from 'history';
 
-import "./admin/databases/databases.module";
-import "./admin/people/people.module";
-import "./admin/settings/settings.module";
-import "./admin/datamodel/datamodel.module";
+import Navbar from "./components/Navbar.jsx";
+import Routes from "./Routes.jsx";
 
+import auth from "metabase/auth/auth";
+import dashboard from "metabase/dashboard/dashboard";
+import databases from "metabase/admin/databases/database";
+import datamodel from "metabase/admin/datamodel/metadata";
+import * as home from "metabase/home/reducers";
+import labels from "metabase/questions/labels";
+import metadata from "metabase/dashboard/metadata";
+import * as people from "metabase/admin/people/reducers";
+import * as pulse from "metabase/pulse/reducers";
+import * as qb from "metabase/query_builder/reducers";
+import questions from "metabase/questions/questions";
+import settings from "metabase/admin/settings/settings";
+import * as setup from "metabase/setup/reducers";
+import undo from "metabase/questions/undo";
```

Comment:
there are quite a few imports here. my instinct would be to try and sort or group them somehow to make it a bit easier to grok. definitely not blocking but would be quite useful.